test/decode_test.exs

```
@@ -10,156 +11,140 @@ defmodule AvroEx.Decode.Test do
       assert {:ok, nil} = @test_module.decode(schema, avro_message)
     end
 
-    test "boolean" do
-      {:ok, schema} = AvroEx.parse_schema(~S("boolean"))
-      {:ok, true_message} = AvroEx.encode(schema, true)
-      {:ok, false_message} = AvroEx.encode(schema, false)
+    property "boolean" do
+      check all bool <- StreamData.boolean() do
+        {:ok, schema} = AvroEx.parse_schema(~S("boolean"))
+        {:ok, encoded} = AvroEx.encode(schema, bool)
```

I'm not sure Property Testing is the right fit for this package. In fact, I'd like us to overhaul the tests in their entirety to use fixtures generated from the Avro spec, not by the library itself. This form of testing is masking errors with encoding and decoding.

If we encode incorrectly, we'll build our code to decode incorrectly, allowing us to deviate from the spec.